spyder/app/tests/test_mainwindow.py

```diff
@@ -156,7 +156,11 @@ def main_window(request):
 
     # Check if we need to use introspection in a given test
     # (it's faster and less memory consuming not to use it!)
-    use_introspection = request.node.get_marker('use_introspection')
+    try:
+        use_introspection = request.node.get_marker('use_introspection')
+    except AttributeError:
+        use_introspection = False
```

Comment:
What's your pytest version? I mean, this is not necessary with pytest 4.0

Reply:
I had 4.3 and now 4.4, will check again

Reply:
This is a Python2 vs 3 issues apparently. I have 4.4.0  no both envs, but it only fails on my py2 env. So I will leave it for now.

Reply:
Ok, thanks for the clarification.